src/main/java/com/google/sps/servlets/DeleteMemberServlet.java

```diff
@@ -51,6 +55,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String memberToDelete = request.getParameter("member-to-delete");
 
+    // Revoke calendar access for the user
+    String calendarID = (String) familyEntity.getProperty("calendarID");
+
+    if(calendarID != null && !memberToDelete.equals(UserServiceFactory.getUserService().getCurrentUser().getEmail())) {
+        Calendar calendarService = Utils.loadCalendarClient();
+        // Iterate over a list of access rules
+        Acl acl = calendarService.acl().list(calendarID).execute();
+
+        for (AclRule rule : acl.getItems()) {
+            if(rule.getScope().getValue().equals(memberToDelete)) {
+                calendarService.acl().delete(calendarID, rule.getId()).execute();
+            }
+        }
+    }
```

Comment:
This servlet does a few things:
1. Delete member from family entity.
2. Remove member from calendar.
3. Remove member from grocery items assigned to them.

In order to keep it readable, its worth having a modular approach where you define different `private` methods that handle each of them separately, eg. `deleteMemberFromFamily(...)`, `removeMemberFromCalendar(...)`, `removeMemberFromGrocery(...)`. Then invoke each of them from the `doPost` method.

Reply:
Made helper methods for remove from calendar and delete user info, however keeping the delete from family code because I will add logic for throwing a response error there